spinflip/diagonalizer.py

```diff
@@ -89,3 +96,62 @@ def state_evolution_plotter(self, t_lim = 'timescale', resolution=500, quantity
         if savefig == True: 
             plt.tight_layout()
             plt.savefig('evolvedstate.png', dpi=300)
+        
+        f.show()
+
+#generates a multiplot of state evolution plotter output but for different Hamiltonians
+#currently only works for quantity = [...] (dont imagine we will use other quantities)
```

Comment:
What does it mean that the function only works for quantity = [...]?

Reply:
quantity can be a string describing different values like 'lminusr' or it can be a list of flavor components; it only works for the list

Reply:
Ah, I see. Could you just copy and paste that into the comment?